Examples/Example/PINRemoteImage.xcodeproj/project.pbxproj

```diff
@@ -1,474 +0,0 @@
-// !$*UTF8*$!
```

Comment:
I thought it's no longer used, but after revisiting seems it's broken just for me, reverted it back, sorry.